react/__snapshots__/examples.spec.jsx.snap

```diff
@@ -6,7 +6,7 @@ exports[`ActionMenu should render examples: ActionMenu 1`] = `
         <use xlink:href=\\"#bottom\\"></use>
       </svg></button>
     <div>
-      <div style=\\"position: absolute; left: 0px; top: 0px; z-index: #fff; transform: translate(0px, 0px);\\" data-popper-placement=\\"bottom-start\\">
+      <div style=\\"position: absolute; left: 0px; top: 0px; z-index: #fff;\\">
```

Comment:
there is a pending issue with the popper component. Don't try to fix it in your PR, even if the CI is red